src/Shared/Recruit.Vacancies.Client/Domain/Entities/VacancyReview.cs

```diff
@@ -30,7 +30,6 @@ public class VacancyReview
         public string ManualQaComment { get; set; }
         public List<ManualQaFieldIndicator> ManualQaFieldIndicators { get; set; }
         public string PrivateReviewNotes { get; set; }
```

Comment:
Not against the dropping of this field if you have found no code relies on it but could you give a heads up in the future regarding this as it has implications on queries used for investigation purposes.

Reply:
What has Mark got to do with it